app/src/main/java/crux/bphc/cms/fragments/ForumFragment.kt

```diff
@@ -113,6 +113,17 @@ class ForumFragment : Fragment() {
                 realm.close()
 
                 CoroutineScope(Dispatchers.Main).launch {
+
+                    if (discussions.size == 0) {
+                        if(forumId != 1) emptyView.text = getString(R.string.no_announcements)
+                        else emptyView.text = getString(R.string.no_posts_to_display)
```

Comment:
Will an `if` expression look better?
`emptyView.text = getString(R.string.no_announcements) if forumId != 1 else getString(R.string.no_posts_to_display)`

Reply:
Point taken, I'll make the change